src/D2L.CodeStyle.TestAnalyzers/Common/Diagnostics.cs

```diff
@@ -2,25 +2,6 @@
 
 namespace D2L.CodeStyle.TestAnalyzers.Common {
 	public static class Diagnostics {
-		public static readonly DiagnosticDescriptor TestCaseSourceStrings = new DiagnosticDescriptor(

```

Comment:
Could you leave comments describing what these were? e.g.:

https://github.com/Brightspace/D2L.CodeStyle/blob/babbc3204f747d6543f107fe4c44f82f0f5f42d5/src/D2L.CodeStyle.Analyzers/Diagnostics.cs#L39-L41